vllm/outputs.py

```diff
@@ -71,6 +77,11 @@ def __init__(
         prompt_logprobs: Optional[PromptLogprobs],
         outputs: List[CompletionOutput],
         finished: bool,
+        arrival_time: float,
+        first_scheduled_time: float,
+        first_token_time: float,
+        time_in_queue: float,
+        finished_time: Optional[float] = None,
```

Comment:
Would it make sense to group these in a separate `RequestMetrics` dataclass?

Reply:
Agree, I feel it might be better to put them in the RequestMetrics class and mark it as Optional.